ns1/ipam.py

```diff
@@ -1050,12 +1053,45 @@ def update(self, address_family, options, server_options=None):
             self.server_options = server_options
 
         self.address_family = address_family
-        self.__dict__.update(options)
+        self.options = options
         self.__dict__.update(server_options)
+
         self.option_list = {
             "options": [
-                {"name": "%s/%s" % (self.address_family, key), "value": value}
-                for key, value in options.items()
+                option.generate_option(address_family)
+                for option in self.options
             ]
```

Comment:
Put this all on one line starting with "options".

Reply:
Originally it was in one line but I got problem with lint check, so I created these two lines implementation.

Reply:
It’s a bit odd, but I suppose if black did this because of the 80 character limit, then it’s ok.

Reply:
black seems to really like splitting comprehensions like this even when it's under 80 chars and to my suprise i, at least, don't even mind.

Reply:
Solved